src/util/CourseUtil.tsx

```diff
@@ -1,52 +1,56 @@
-import { useSelector } from 'react-redux';
-
 import { Author } from 'src/components/Courses/components/CourseCard/Author.types';
 import { CourseCardData } from 'src/components/Courses/components/CourseCard';
 import { Course } from 'src/course.type';
-import { RootState } from 'src/store';
 
-export const buildAuthor = (id: string, name: string): Author => {
-	return { id: id, name: name };
-};
+export class CourseUtil {
+	public static buildAuthor = (id: string, name: string): Author => {
```

Comment:
for utils in js we usually use just common functions, and for linked and more difficult logic use services (it's about naming)